### PR TITLE
explorer: Add accessible name to Nav Bar button

### LINKED ISSUE #2036: explorer: Nav button does not have an accessible name
Screenshot 2024-07-26 at 15 15 13

### PR DESCRIPTION
**Pull request by nortonandreev**
Resolves #2036

### CODE REVIEW

File: explorer/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add accessible name to the nav bar button on mobile [#2036]
```
Comment (ascartabelli) [inline explorer/CHANGELOG.md:12]: Why "on mobile"?

**Review by ascartabelli — APPROVED** [review]
Good to merge, but the changelog entry seems a bit odd.

File: explorer/CHANGELOG.md (hunk 1 above)
Comment (nortonandreev) [reply explorer/CHANGELOG.md:12]: We can get rid of it - the idea is that, from the user's perspective, the nav bar toggle is only visible on mobile.

Comment (ascartabelli) [reply explorer/CHANGELOG.md:12]: ah, ok, leave it then

**Review by ascartabelli — APPROVED** [review]

**Pull request merged**